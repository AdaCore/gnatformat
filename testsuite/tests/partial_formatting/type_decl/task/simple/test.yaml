description: TaskTypeDecl formatting for line selection.
driver: partial_gnatformat
args:
  [
    "test.ads",
    "-SL 115",
    "-EL 116",
  ]
status_code: 1