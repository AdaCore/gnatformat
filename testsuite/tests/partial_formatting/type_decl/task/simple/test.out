<CompilationUnit test.ads:27:1-149:23>
test.ads(<TaskTypeDecl ["Server"] test.ads:115:4-118:15>) - 115:1-118:15
^
   task type Server
   is
      entry Next_Work_Item (WI : in Work_Item);
      entry Shut_Down;
   end Server;$

